### PR TITLE
Remove kernel-modules-extra dependency

### PR DESCRIPTION
**Pull request by pgeng-nv**
RM #3853731

### CODE REVIEW

File: rshim.spec.in [hunk 1]
```diff
@@ -15,9 +15,7 @@ Source0: https://github.com/mellanox/rshim-user-space/archive/%{name}-%{version}
 BuildRequires: gcc, autoconf, automake, pkgconfig, make
 BuildRequires: pkgconfig(libpci), pkgconfig(libusb-1.0)
 
-%if (0%{?rhel} >= 8 || 0%{?fedora} > 0) && "0%{?ctyunos}" == "0"
-Requires: kernel-modules-extra
-%endif
+Requires: kmod(cuse.ko)
```
Comment (lsun100) [inline rshim.spec.in:18]: I am wondering whether this syntax works in all our existing BFBs.  ++@vladsokolovsky in case he might have some comments.

Comment (pgeng-nv) [reply rshim.spec.in:18]: Good comment. Let's hold-off on this change. Let me test it on different distributions we need to support and see whether it works.

Comment (lsun100) [reply rshim.spec.in:18]: That'll be great. It's not related to BFB, but need to try different hosts (at least centos 7.x)

**Comment by pgeng-nv** [comment]
Will close this as cuse.ko still depends on rpm package of kernel-modules-extra on RHEL 9, so it doesn't solve the RM for those cases. The dependency is unavoidable, so the RM has been rejected.

**Pull request closed without merging**